lib/experimental/PageLayouts/Utils/DetailsItemsList/index.stories.tsx

```diff
@@ -36,7 +36,7 @@ const meta: Meta = {
         title: "Workable days",
         content: {
           type: "weekdays",
-          activatedDays: ["Monday", "Tuesday", "Wednesday"],
+          activatedDays: [0, 1, 2],
```

Comment:
Why not using the Date standard were 0 is sunday?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay

Reply:
the current solution is unaware it shows dates. Technically, it shows a list of strings and we pick values from this list according indices passed. 

The next step in the evolution of this component will be making it date aware and use the standard